react/SelectBox/Readme.md

```diff
@@ -279,3 +279,83 @@ const options = [
 
 <SelectBox options={options} menuIsOpen classNamePrefix='needsclick cz' />
 ```
+
+### SelectBox in Dialog
+
```

Comment:
Can you add a sentence on the need to use "fixed" ?

Reply:
Com'on, "Selectbox in dialog" is not understandable? 🤣 I added a reproductible exemple of my issue to fix it at first  without knowing what the issue was before. Now that I know it, i'll add a description. 

3f33a53f